packages/admin/src/index.ts

```diff
@@ -126,3 +126,9 @@ export { ErrorDialogProvider } from "./error/errordialog/ErrorDialogProvider";
 export { useErrorDialog } from "./error/errordialog/useErrorDialog";
 export { ErrorScope, LocalErrorScopeApolloContext, errorScopeForOperationContext } from "./error/errordialog/ErrorScope";
 export { ErrorDialogApolloLinkOptions, createErrorDialogApolloLink } from "./error/errordialog/createErrorDialogApolloLink";
+export { SplitButton, SplitButtonProps } from "./common/buttons/split/SplitButton";
+export { CometAdminSplitButtonThemeProps } from "./common/buttons/split/SplitButton.styles";
+export { SaveButton, SaveButtonProps, DisplayStateSaveButton } from "./common/buttons/save/SaveButton";
```

Comment:
now that we have the public api nicely presented we can discuss naming for new exports: I'd suggest to rename `DisplayStateSaveButton` to `SaveButtonDisplayState`, to have a fixed "module" prefix

Reply:
if we don't need it we should not export it!